README.md

```diff
@@ -1 +1,18 @@
-﻿# Balanced-Management
+﻿# Automated-Compliance
+
+I'm a college student working on a complex project related to regulatory compliance at an insurance company.
```

Comment:
Rispondo in italiano, se non c'è problema. E dico 
🤌  🤌  e ancora 🤌 
Mi racommando, leggi [la guida](http://jj.github.io/IV/documentos/proyecto/0.Repositorio#tldr):

> En cuanto a la idea o problema que se quiere resolver, tiene que ser un problema del cual tengas conocimiento personal, preferiblemente. Cuanto más involucrado estés con el problema, o las personas que lo tienen, más interés vas a tener en hallar una solución al mismo y mejor entenderás qué sería lo que lo soluciona.

E un tuo problema la ditta di siccuranza? Non metti "sono un studente" perche comunque non è il tuo problema.